Classes/Issues/IssuesViewController.swift

```diff
@@ -234,7 +234,7 @@ IssueCommentSectionControllerDelegate {
             status != .merged
             else { return nil }
         
-        return AlertAction.toggleIssue(status) { [weak self] _ in
+        return AlertAction.toggleIssue(status, issue: current?.status.pullRequest != true) { [weak self] _ in
```

Comment:
Wait, do we not have access to the `NotificationType` from here?

Reply:
Pretty sure that's only available from the Notification level, not passed through once you go into an issue. This could be simplified slightly, in hindsight, by just doing `current?.pullRequest` but yh